### PR TITLE
feat(procfile): unskip builds procfile test

### PR DESCRIPTION
**Pull request by smothiki**
Previously the test is not waiting for the commands to finish execution add exit(o) gomega function to wait for the command to finish executing 

closes https://github.com/deis/workflow-e2e/issues/299

### CODE REVIEW

File: tests/buildprocfile_test.go [hunk 1]
```diff
@@ -41,27 +40,28 @@ var _ = Describe("deis builds procfile", func() {
 			})
 			// https://github.com/deis/workflow-e2e/issues/299
 			// TODO(smothiki): somehow determine why this test is flaky in jenkins after merging deployments
```
Comment (helgi) [inline tests/buildprocfile_test.go]: want to remove this as well, if this resolve things

Comment (smothiki) [inline tests/buildprocfile_test.go]: Oops will remove that once CI passes tests

File: tests/buildprocfile_test.go [hunk 2]
```diff
@@ -39,29 +38,27 @@ var _ = Describe("deis builds procfile", func() {
 			AfterEach(func() {
 				apps.Destroy(user, app)
 			})
-			// https://github.com/deis/workflow-e2e/issues/299
-			// TODO(smothiki): somehow determine why this test is flaky in jenkins after merging deployments
-			XSpecify("that user create a new build of that app with a different procfile", func() {
-				// Docker Hub gives a "not found" 400 error
+			Specify("that user create a new build of that app with a different procfile", func() {
```
Comment (vdice) [inline tests/buildprocfile_test.go]: minor nit on the text here: `"that user can create ..."` would read better

Comment (smothiki) [inline tests/buildprocfile_test.go]: sure

**Pull request merged**